calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_calibrator.py

```diff
@@ -950,11 +1103,33 @@ def process_detection_results(self, img: np.array, detection: BoardDetection, im
                 f"Reprojection error (rms): {reprojection_error_rms:.3f} px ({100.0 * reprojection_error_rms_relative:.2f}%)"  # noqa E231
             )
 
-            self.training_occupancy_rate_label.setText(
-                f"Training occupancy: {100.0*self.data_collector.get_training_occupancy_rate():.2f}"  # noqa E231
+            if self.operation_mode == OperationMode.CALIBRATION:
+                self.training_occupancy_rate_label.setText(
+                    f"Training occupancy: {100.0*self.data_collector.get_training_occupancy_rate():.2f}"  # noqa E231
+                )
+                self.evaluation_occupancy_rate_label.setText(
+                    f"Evaluation occupancy: {100.0*self.data_collector.get_evaluation_occupancy_rate():.2f}"  # noqa E231
+                )
+
+            board_speed = (
+                0 if self.last_detection is None else detection.get_speed(self.last_detection)
```

Comment:
should not we set the speed to a high value for the first detection ?

Reply:
you're right, thanks, i will fix it

Reply:
This commit does not have the changes.
If the value is a floating point, you should not use 100

Reply:
[ec063e9](https://github.com/tier4/CalibrationTools/pull/220/commits/ec063e916576efd4d43021693ba83ffc22ae1910)